creator-node/default-config.json

```diff
@@ -15,6 +15,7 @@
   "rateLimitingMetadataReqLimit": 3000,
   "rateLimitingImageReqLimit": 6000,
   "rateLimitingTrackReqLimit": 6000,
+  "rateLimitingBatchCidsExistLimit": 6000,
```

Comment:
would say 6000 is way way too high of a limit, lets make this something very small like 600, realistically whatever your script needs and nothing higher

Reply:
I think thousand would be a better estimate considering that we might rerun a couple times locally.

Reply:
for sure - but 1000 * 500 = 500,000 FS reads per hour on prod nodes could be pretty heavy. we should just rate limit the script accordingly. lets set this number together